java/ql/src/Security/CWE/CWE-1004/InsecureTomcatConfig.qhelp

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE qhelp SYSTEM "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>When you add an application to tomcat server, it will generate a new JSESSIONID when you call `request.getSession()` 
```

Comment:
A couple of small formatting suggestions:
```suggestion
<p>When you add an application to a Tomcat server, it will generate a new <code>JSESSIONID</code> when you call <code>request.getSession()</code> 
```